python/ciqueue/distributed.py

```diff
@@ -38,7 +40,7 @@ def wait_for_master(self, timeout=10):
             "` after {} seconds waiting.".format(timeout))
 
     def _master_status(self):
-        return self.redis.get(self.key('master-status'))
+        return self.redis.get(self.key('master-status')).decode()
```

Comment:
Hm, do we really have to call `.decode` anytime we read from redis? Is there a setting that can force redis to just always return a string?

Reply:
I'll check.

Reply:
There is such a setting, but that causes failures if any of your values are _not_ unicode strings. CIQueue, as it happens, stores pickled Python objects as binary data.